src/aoapplication.cpp

```diff
@@ -171,6 +173,12 @@ void AOApplication::ms_connect_finished(bool connected, bool will_retry)
   }
 }
 
+void AOApplication::skip_timers(qint64 msecs)
+{
+  if (courtroom_constructed)
+    w_courtroom->skip_clocks(msecs);
+}
+
```

Comment:
Why not connect skip_timers directly from the demo server to the courtroom when AOApplication is constructing the courtroom?